docs/advanced/how-to-integrate-scrypt-service.md

```diff
@@ -0,0 +1,64 @@
+---
+sidebar_position: 5
+---
+
+# How to Integrate sCrypt Service
+
+## Initialize sCrypt Client
+
+Passing your own API key to the `Scrypt.init` function to initialize the sCrypt client. 
+
+```ts
+import { Scrypt } from 'scrypt-ts'
+
+Scrypt.init({
+  apiKey: 'YOUR_API_KEY',
+})
+```
+
+:::note
+For now, you can use the test key `alpha_test_api_key`.
+:::
+
+## Upload Contract Artifact
+
+Before deploying the contract, it's required to upload the contract artifact.
+
+```ts
+await DemoContract.compile()
+
+// need to upload contract artifact before deploying it
+const hexHash = await Scrypt.contractApi.uploadArtifact(DemoContract)
```

Comment:
This should take json as input, not necessarily a contract https://scrypt.io/docs/how-to-integrate-a-frontend#1-compile-contract